python-lib/gluonts_forecasts/training_session.py

```diff
@@ -83,7 +83,7 @@ def init(self, session_name, partition_root=None):
             partition_root (str, optional): Partition root path, concatenated to session_name to create the session_path. Defaults to None.
         """
         self.session_name = session_name
-        if partition_root is None:
+        if not partition_root:
```

Comment:
Will fail if partition_root == 0, should use 'is None'.

Reply:
```suggestion
        if partition_root is None:
```